pyacq/core/rpc/tests/test_rpc.py

```diff
@@ -0,0 +1,347 @@
+import threading, atexit, time, logging
```

Comment:
I get this on my machine:
[22m[32m[samuel-HP-Z800-Workstation:main_process:thread-7f730ec25700][0m [1m[33mFailed to call object <bound method TestClass.add of <__main__.test_rpc.<locals>.TestClass object at 0x7f731a953e10>>: 2, ('x',)[0m
[22m[32m[samuel-HP-Z800-Workstation:main_process:thread-7f730ec25700][0m [1m[33m    => returning exception for 18: (<class 'TypeError'>, TypeError("unsupported operand type(s) for +: 'int' and 'str'",), <traceback object at 0x7f73197fc1c8>)[0m
Traceback (most recent call last):
  File "test_rpc.py", line 348, in <module>
    test_disconnect() 
  File "test_rpc.py", line 312, in test_disconnect
    server_proc = ProcessSpawner()
  File "/home/samuel/Documents/projet/pyacq-0.2/pyacq/core/rpc/processspawner.py", line 164, in __init__
    status = bootstrap_sock.recv_json()
  File "/usr/lib/python3/dist-packages/zmq/sugar/socket.py", line 412, in recv_json
    msg = self.recv(flags)
  File "zmq/backend/cython/socket.pyx", line 631, in zmq.backend.cython.socket.Socket.recv (zmq/backend/cython/socket.c:5799)
  File "zmq/backend/cython/socket.pyx", line 665, in zmq.backend.cython.socket.Socket.recv (zmq/backend/cython/socket.c:5599)
  File "zmq/backend/cython/socket.pyx", line 139, in zmq.backend.cython.socket._recv_copy (zmq/backend/cython/socket.c:1752)
  File "zmq/backend/cython/checkrc.pxd", line 21, in zmq.backend.cython.checkrc._check_rc (zmq/backend/cython/socket.c:6275)
zmq.error.ZMQError: Appel système interrompu
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/home/samuel/Documents/projet/pyacq-0.2/pyacq/core/rpc/processspawner.py", line 188, in stop
    closed = self.client.close_server()
AttributeError: 'ProcessSpawner' object has no attribute 'client'